docs/rpc_nodes.md

```diff
@@ -17,17 +17,17 @@ author: Roxane Letourneau
 
 - Tezos Giga Node from Tezos Ukraine
     - Mainnet: https://mainnet-tezos.giganode.io
-    - Edo testnet: https://edonet-tezos.giganode.io
+    - Granada testnet: https://granadanet-tezos.giganode.io
```

Comment:
https://granadanet-tezos.giganode.io is not a valid URL

https://testnet-tezos.giganode.io/

Reply:
fixed